docker/docker-compose.yml

```diff
@@ -0,0 +1,42 @@
+name: cdash
```

Comment:
Did you try using `docker compose` instead of `docker-compose`?   e.g., `docker compose -f docker/docker-compose.yml -f docker/docker-compose.mysql.yml -f docker/docker-compose.dev.yml --env-file .env.dev up -d`

Reply:
This was resolved by moving to a proper install of `docker-engine` and the compose plugin.